grass7/imagery/i.sentinel/i.sentinel.mask/i.sentinel.mask.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
```suggestion
#!/usr/bin/env python
```
g.extension will take care of this